modules/mini-games/tightspot.lua

```diff
@@ -0,0 +1,758 @@
+            end
+        end
+    else
+        local time = variables.level.play_time
+        time = SecondsToClock((time - (game.tick - variables.tick))/60)
+        if time == "00:00" then
+            Mini_games.stop_game()
+        end
+        for i,player in ipairs(game.connected_players) do
+            local Main_gui = Gui.get_left_element(player, game_gui)
+            local table = Main_gui.container["Money"].table
+            table["Time"].caption = time
+        end
+    end
+end
+
+token_for_replace_wall =  Token.register(replace_wall)
+Store.watch(balances,function(value,key,_)
+    local player = game.players[key]
+    local Main_gui = Gui.get_left_element(player, game_gui)
+    local table = Main_gui.container["Money"].table
+    table["Balance"].caption = value
+end)
+
+
+local function market(event)
+    local player = game.players[event.player_index]
+    Store.set(balances,player,player.get_item_count("coin"))
+end
+
+local function insideBox(box, pos)
+    local x1 = box[1][1]
+    local y1 = box[1][2]
+    local x2 = box[2][1]
+    local y2 = box[2][2]
+
+    local px = pos.x
+    local py = pos.y
+    return px >= x1 and px <= x2 and py >= y1 and py <= y2
+end
+
+local function player_move(event)
+    local player = game.players[event.player_index]
+    if player.surface.name == variables.level.surface then  --check if the player has not been tped away
+        local center = centers[player.name]
+        if center then
+            local pos = player.position
+            local area = islands[player.name]
+            if not insideBox(area,pos) then
+                player.teleport({center.x,center.y},variables.level.surface)
+            end
+        end
+    end
+end
+
+local function on_player_left_game(event)
+    local player = game.players[event.player_index]
+
+    local inv = player.get_inventory(defines.inventory.god_main)
+    local all_items = inv.get_contents()
+    game.print(serpent.block(all_items))
+    left_players[player.name] = all_items
+
+    player.set_controller {type = defines.controllers.god}
+    player.create_character()
+    player.teleport({-35,55},"nauvis")
+    --gui
+    local Main_gui = Gui.get_left_element(player, game_gui)
+    local table = Main_gui.container["slider"].table
+    table.visible = false
+    Gui.toggle_left_element(player,  Main_gui, false)
+end
+
+local function player_join(event)
+    local player = game.players[event.player_index]
+    player.character.destroy()
+    local items = left_players[player.name]
+    local center = centers[player.name]
+    if items then
+        player.teleport(center,variables.level.surface)
+        if started[1] then
+            player.set_controller {type = defines.controllers.spectator}
+        else
+            player.set_controller {type = defines.controllers.god}
+            local Main_gui = Gui.get_left_element(player, game_gui)
+            Gui.toggle_left_element(player,  Main_gui, true)
```

Comment:
This will not work, this applies all the times you use it.
Gui.toggle_left_element takes an element define not an element.
This should be `Gui.toggle_left_element(player, game_gui, true)`

Reply:
wierd as it does work 😛